package.json

```diff
@@ -30,7 +30,7 @@
     "uuid": "^3.3.2"
   },
   "scripts": {
-    "start": "chmod +x ./init.sh && ./init.sh && cp env-config.js ./public/ && npm run build:css && react-scripts start",
```

Comment:
How is this related to the ticket?

Reply:
I have created a separate ticket for it. I'll remove the change from this one